### PR TITLE
Possible fix for issue 75: null user names breaking things.

### PR DESCRIPTION
**Pull request by hackmud-dtr**

### CODE REVIEW

File: js/ui.js [hunk 1]
```diff
@@ -224,6 +224,7 @@ function replaceUI() {
 }
 
 function colorizeUser(user) {
+	if(!user)return "`C[deleted user]`";
```
Comment (seanmakesgames) [inline js/ui.js]: I'd like this to be consistent with the forums. They are currently
`\`C<user deleted>\``
Which is preferred?

Comment (NGTmeaty) [reply js/ui.js]: <user deleted> looks better to me. also on the forums so 👍

Comment (seanmakesgames) [reply js/ui.js]: Let's change it to `\`C<user deleted>\``

Comment (hackmud-dtr) [reply js/ui.js]: @seanmakesgames I believe I have this change in now. Sorry didn't see this.

Comment (seanmakesgames) [reply js/ui.js]: no probs! Thanks for changing. :)

**Pull request merged**